### ISSUE TITLE
The latest template in master branch does not work

### ISSUE THREAD

**Issue by LalatenduMohanty** [redhat-developer/s2i-dotnetcore#164]
I am trying to run the `dotnet-runtime-example-runtime` on top of Minishift using the dotnet add-on which downloads the templates and image streams from here. It seems the templates are not working as expected and giving me following error.

```
An error occurred while starting the build.
Reason: Error resolving ImageStreamTag dotnet:2.0 in namespace openshift: unable to find latest tagged image
```

[1] https://github.com/minishift/minishift-addons/tree/master/add-ons/dotnet

**Comment by aslicerh** [comment]
Hey @LalatenduMohanty, I haven't tried this on minishift before (working on that now), but I'm not having trouble with the imagestream directly on a local openshift install.

Can you run commands directly in the VM created by minishift? If so, can you see if you can pull down the image defined in the imagestream directly by running:
```
docker pull registry.access.redhat.com/dotnet/dotnet-20-runtime-rhel7:2.0
```

**Comment by gbraad** [comment]
I also do not see this issue... are you sure the imagestream got downloaded?

```
{
                        "name": "2.0",
                        "annotations": {
                            "openshift.io/display-name": ".NET Core 2.0",
                            "description": "Build and run .NET Core 2.0 applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/redhat-developer/s2i-dotnetcore/tree/master/2.0/build/README.md.",
                            "iconClass": "icon-dotnet",
                            "tags": "builder,.net,dotnet,dotnetcore,rh-dotnet20",
                            "supports":"dotnet:2.0,dotnet",
                            "sampleRepo": "https://github.com/redhat-developer/s2i-dotnetcore-ex.git",
                            "sampleContextDir": "app",
                            "sampleRef": "dotnetcore-2.0",
                            "version": "2.0"
                        },
                        "from": {
                          "kind": "DockerImage",
                          "name": "registry.access.redhat.com/dotnet/dotnet-20-rhel7:2.0"
                        }
                    },
```

This is included in: https://raw.githubusercontent.com/redhat-developer/s2i-dotnetcore/master/dotnet_imagestreams.json

**Comment by gbraad** [comment]
the image was updated 4 days ago, but the location is still the same

https://access.redhat.com/containers/?tab=overview#/registry.access.redhat.com/dotnet/dotnet-20-rhel7

**Comment by tmds** [comment]
@LalatenduMohanty it is not clear for me if there is still an issue. If there isn't, please close. Otherwise, can you provide a set of instructions to reproduce it? I tried but failed to use the dotnet add-on. minishift's developer user didn't seem to have permissions to add stuff to the openshift namespace.

**Comment by LalatenduMohanty** [comment]
This got resolved with Origin v3.9.0 release. Hence closing this.
